plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -238,7 +238,9 @@ fn main() {
 
     let apt = match AptCli::try_parse() {
         Ok(aptcli) => aptcli,
-        Err(_e) => {
+        Err(err) => {
+            eprintln!("ERROR: {}", err);
+            AptCli::command().print_help().unwrap();
```

Comment:
Remove unwrap

Reply:
Replaced with `expect()` that prints a better error message than `unwrap()`.